pubsub/topics/publish_settings.go

```diff
@@ -35,9 +33,8 @@ func publishWithSettings(w io.Writer, projectID, topicID string) error {
 	if err != nil {
 		return fmt.Errorf("pubsub.NewClient: %v", err)
 	}
-
-	var wg sync.WaitGroup
-	var totalErrors uint64
+	var resultSlice []*pubsub.PublishResult
+	var errors []error
```

Comment:
`errors` is the name of a standard library package (http://pkg.go.dev/errors). So, maybe `resultErrors`?

Reply:
Right!
Done